olm/quickstarts/rhosak-openshift-nodejs-bind-quickstart.yaml

```diff
@@ -0,0 +1,357 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: rhosak-openshift-nodejs-bind-console-quickstart
+spec:
+  displayName: Binding an application to Red Hat OpenShift Streams for Apache Kafka using the OpenShift web console
```

Comment:
Would it make sense to include Node.js in the title so that a Node.js dev knows it might be of interest?

Reply:
Good call. Added this.